src/main/java/org/fundacionjala/at13/spaceinvaders/Space.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.at13.spaceinvaders;
+
+public class Space{
+
+    private static final int SPACE_SIZE_HIGHT=50;
+    private static final int SPACE_SIZE_WIDTH=100;
```

Comment:
Space is missing between "=", in line 5 and 6.

Reply:
ok, sorry!!!